.travis.yml

```diff
@@ -86,6 +86,7 @@ before_install:
 
 install:
   - git fetch -q
+  - git lfs pull
```

Comment:
Any reason for adding that here globally? Wouldn't it be enough to add that to `tests\travis\install_git_lfs.sh`
Guess there is no use in doing that for non UI tests.

Reply:
Updated.